c-pallets/segment-book/src/lib.rs

```diff
@@ -523,6 +533,7 @@ pub mod pallet {
 			Ok(())
 		}
 
+		// Seems ut is unifinished?
```

Comment:
This is an unsigned transaction. I'm not sure if I need to calculate weight for it

Reply:
These are the functions of the current transaction. What good optimization suggestions do you have